src/smc-hub/client.coffee

```diff
@@ -1431,7 +1431,10 @@ class exports.Client extends EventEmitter
                         listing = x; cb(err)
         ], (err) =>
             if err
-                @error_to_client(id:mesg.id, error:err)
+                if misc.startswith(err, 'Traceback')
```

Comment:
I looked at catching the error at lower level & it looked as if refactoring might be needed. This looked like the lower risk option.